backend/test/infra/controller/TransactionController.test.ts

```diff
@@ -81,32 +81,32 @@ describe("TransactionController", () => {
           id: 3,
           name: "new group",
           members: [new GroupMember(1, "John")],
-          transactions: [new Transaction(0, 1, 3)],
+          transactions: [new Transaction(0, 1, [1, 2], 3)],
         },
       });
 
       // Act
       const response = await server.inject({
         method: "POST",
-        url: `/group/${3}/transaction`,
-        payload: { payerId: 0, amount: 1 },
+        url: `/group/3/transaction`,
```

Comment:
Would be nice to not have raw strings/numbers (by creating the objects beforhand) so you can reference these
=> It would make the tests easier to read